sequencer/src/header.rs

```diff
@@ -203,6 +210,7 @@ impl Header {
         let fee_merkle_tree_root = state.fee_merkle_tree.commitment();
 
         let header = Self {
+            chain_config: chain_config.into(),
```

Comment:
Should this be `chain_config.commit().into()`? Otherwise we're including the full chain config in every header

Reply:
we have that defined on the `Header` as `chain_config: ResolvableChainConfig`. should that be changed to `Commitment<ChainConfig>`?

Reply:
My understanding of the docs is that the Header field can be either the full `ChainConfig` or the `Commitment`. As it is not that `into()` will resolve to `ResolvableChainConfig` which holds an `Either` wrapping those two things.

Reply:
Ah sorry, maybe I wasn't clear in the docs. It can indeed be either, but we mainly only want the full thing in the genesis. After that it's better to include just a commitment to keep the headers small (this will benefit more as we add more fields to it)